.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
```

Comment:
What does that mean? Even internally we can't make any other issues apart from bug, feature and question anymore?

Reply:
It's mainly for external users without repository write access. On my test repository I also had a test project; I was able to convert a draft issue into a repository issue from the project board without issue even though blank issues enabled was false. And failing that, you can always go to https://github.com/aica-technology/api/issues/new (instead of https://github.com/aica-technology/api/issues/new/choose) to create a new blank issue